content/rs/_index.md

```diff
@@ -13,6 +13,106 @@ aliases: /rs/
          /redis_enterprise_software/
 ---
 
+
+[Redis Enterprise](https://redislabs.com/redis-enterprise/) is a robust, in-memory database platform built by the same people who develop open source Redis.
+It maintains the simplicity and high performance of Redis and adds many enterprise-grade capabilities, including:
+
+- Linear scaling to hundreds of millions of operations per second
+- Active-Active global distribution with local latency
+- Redis on Flash to support large datasets while minimizing infrastructure costs
+- 99.999% uptime
+
+## Software vs. Cloud
+
+Redis Enterprise is available as software and as a hosted [cloud service]({{< relref "/rc/_index.md" >}}).
+
+You can run Redis Enterprise Software in an on-premises data center or on your preferred cloud platform. 
+
+Redis Enterprise also works with [Kubernetes]({{< relref "/kubernetes/_index.md" >}}) and [Openshift]({{< relref "/kubernetes/openshift/_index.md" >}}).
+
+For development and testing, you can also run Redis Enterprise using [Docker containers]({{< relref "/rs/installing-upgrading/get-started-docker.md" >}}).
+
+If you prefer a managed version of Redis Enterprise, [Redis Enterprise Cloud]({{< relref "/rc/_index.md" >}}) provides all of the benefits of Redis Enterprise Software on all major public clouds.
+
+## Why Redis Enterprise Software?
+
+Redis Enterprise makes it easy to run Redis at scale.
+You can use a single Redis Enterprise Cluster to provision as many Redis databases as you need for your organization, and each of these databases can be tuned and scaled independently.
+In addition, Redis Enterprise supports many enterprise security features, including role-based access control.
+
+For more on scaling, high availability, and durability, see the [the Redis Enterprise Software architecture.]({{< relref "/rs/concepts/_index.md" >}})
+
+## Active-Active replicated Redis databases (CRDB) {#activeactive-replicated-redis-databases-crdb}
+
+[Active-Active databases]({{< relref "/rs/databases/active-active/" >}}) provide global, geo-distributed database replication with conflict-free local writes.
+This greatly simplifies the development of globally distributed applications.
+Geo-distributed Active-Active databases combine [conflict-free replicated data types](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) with Redis data types to provide automatic conflict resolution based on each data type's semantics.
```

Comment:
How do they achieve automatic conflict resolution?

Reply:
The database does it automatically during the syncing process.

Reply:
The part "based on each data type's semantics" intrigued me. What role do they play and how?